frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/AccountInterventionsHandlerTest.java

```diff
@@ -2,21 +2,251 @@
+            throws Json.JsonException, UnsuccessfulAccountInterventionsResponseException {
+        var context = mock(Context.class);
+        var event = new APIGatewayProxyRequestEvent();
+        event.setHeaders(getHeaders());
+        event.setBody(format("{ \"email\": \"%s\" }", TEST_EMAIL_ADDRESS));
+        when(authenticationService.getUserProfileByEmailMaybe(anyString()))
+                .thenReturn(Optional.of(generateUserProfile()));
+        when(accountInterventionsService.sendAccountInterventionsOutboundRequest(any()))
+                .thenThrow(
+                        new UnsuccessfulAccountInterventionsResponseException(
+                                "Gateway timeout error", 504));
+        var result = handler.handleRequest(event, context);
+        assertThat(result, hasStatus(504));
+        assertThat(result, hasBody(objectMapper.writeValueAsString(ErrorResponse.ERROR_1054)));
+    }
+
+    @Test
+    void shouldReturnErrorResponseWhenAccountInterventionsAPIReturnsAnyOtherHttpStatusCode()
+            throws Json.JsonException, UnsuccessfulAccountInterventionsResponseException {
+        var context = mock(Context.class);
+        var event = new APIGatewayProxyRequestEvent();
+        event.setHeaders(getHeaders());
+        event.setBody(format("{ \"email\": \"%s\" }", TEST_EMAIL_ADDRESS));
+        when(authenticationService.getUserProfileByEmailMaybe(anyString()))
+                .thenReturn(Optional.of(generateUserProfile()));
+        when(accountInterventionsService.sendAccountInterventionsOutboundRequest(any()))
+                .thenThrow(
+                        new UnsuccessfulAccountInterventionsResponseException(
+                                "Unexpected error", 404));
+        var result = handler.handleRequest(event, context);
+        assertThat(result, hasStatus(404));
+        assertThat(result, hasBody(objectMapper.writeValueAsString(ErrorResponse.ERROR_1055)));
+    }
+
+    @Test
+    void shouldReturn200ForSuccessfulRequest()
+            throws UnsuccessfulAccountInterventionsResponseException, Json.JsonException {
+        var context = mock(Context.class);
+        var event = new APIGatewayProxyRequestEvent();
+        event.setHeaders(getHeaders());
+        event.setBody(format("{ \"email\": \"%s\" }", TEST_EMAIL_ADDRESS));
+        when(authenticationService.getUserProfileByEmailMaybe(anyString()))
+                .thenReturn(Optional.of(generateUserProfile()));
+        when(accountInterventionsService.sendAccountInterventionsOutboundRequest(any()))
+                .thenReturn(generateAccountInterventionResponse());
+        var result = handler.handleRequest(event, context);
+        assertThat(result, hasStatus(200));
+
+        var accountInterventionsResponse = new AccountInterventionsResponse(true, false, false);
+        assertThat(result, hasBody(objectMapper.writeValueAsString(accountInterventionsResponse)));
```

Comment:
This looks to me locally like it's passing for the wrong reason - both the result body and the `objectMapper.writeValueAsString(accountInterventionsResponse)` are returning empty json objects. In general, I'd prefer testing against literal json strings to make sure we can't get tests incorrectly passing like this